tests/zfs-tests/tests/functional/devices/devices_common.kshlib

```diff
@@ -39,6 +39,11 @@
 #
 function create_dev_file
 {
+	if is_freebsd; then
+		create_dev_file_freebsd "$1" "$2"
+		return $?
+	fi
```

Comment:
Similar to above adding a `create_dev_file_linux` would help readability and consistency.